tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/iam/IdentitiesSpec.scala

```diff
@@ -0,0 +1,30 @@
+package ch.epfl.bluebrain.nexus.tests.iam
+
+import akka.http.scaladsl.model.StatusCodes
+import ch.epfl.bluebrain.nexus.tests.HttpClient.tokensMap
+import ch.epfl.bluebrain.nexus.tests.{BaseSpec, Identity}
+import io.circe.Json
+
+class IdentitiesSpec extends BaseSpec {
+
+  "The /identities endpoint" should {
+    s"return identities of the user" in {
+      deltaClient.get[Json]("/identities", Identity.ServiceAccount) { (json, response) =>
+        response.status shouldEqual StatusCodes.OK
+        json shouldEqual jsonContentOf(
+          "/iam/identities/response.json",
+          "deltaUri" -> config.deltaUri.toString()
+        )
+      }
+    }
+
+    "return the error for an invalid token" in {
+      tokensMap.put(Identity.InvalidTokenUser, toAuthorizationHeader("INVALID"))
+
+      deltaClient.get[Json]("/identities", Identity.InvalidTokenUser) { (json, response) =>
+        response.status shouldEqual StatusCodes.Unauthorized
+        json shouldEqual jsonContentOf("/iam/identities/errors.json")
```

Comment:
I wonder whether asserting on the entire JSON makes this too brittle. Perhaps if we assert that the @context and @type are correct, the reason text should be able to change without failing?

Reply:
I wanted to check that `reason` is actually transmitted.
I thought also a bit about this and I chose this one because the reason for this specific error is a constant and we handle it ourselves.
Other token errors include in their reason coming from a third party library that is more likely to change